_posts/2022-11-21-november-demo-day.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: Dask Demo Day November 2022
+author: Richard Pelgrim (Coiled)
+tagline: an optional tagline
+tags: [Community]
+theme: twitter
+draft: true
+---
+
+{% include JB/setup %}
+
+# Dask Demo Day : November
+
+Once a month, the Dask Community team hosts Dask Demo Day: an informal and fun online hangout where folks can showcase new or lesser-known Dask features…and the rest of us can learn about all the things we didn’t know Dask could do 😁
+
+November’s Dask Demo Day had 5 great demo's. We learned about:
+- the Dask-Optuna integration for distributed hyperparameter optimization
+- visualizing 2-billion lightning flashes with Dask, RAPIDS and Datashader
+- the new Dask CLI
+- Dask-Awkward
+- profiling your Dask code with Dask-PySpy 
```

Comment:
Can we add links to all the things here? If there is a blog post or anything on the topic we could link the whole line to that, if not we could just link the words like RAPIDS, Dask-Awkward, etc to their respective project pages.

Reply:
I've added links to each sub-section in the blogpost (and in there are the links to the outside projects, like RAPIDS, dask-awkward, etc.)